### PR TITLE
Lint

### PR DESCRIPTION
**Pull request by fregante**
🥴

### CODE REVIEW

**Comment by fregante** [comment]
There's a comment left for https://github.com/refined-github/refined-github/pull/4982 that would require a lot of related changes, it'd be great to see that cleanup done separately

File: .xo-config.json [hunk 1]
```diff
@@ -15,6 +15,7 @@
 			"allowWarningComments": false
 		}],
 		"@typescript-eslint/naming-convention": "off",
+		"@typescript-eslint/no-redundant-type-constituents": "off",
```
Comment (fregante) [inline .xo-config.json:19]: Due to the wrong `Deinit` type introduced by #5422

File: .xo-config.json [hunk 2]
```diff
@@ -8,13 +8,15 @@
 		"__filebasename"
 	],
 	"rules": {
+		"jsx-quotes": "error",
```
Comment (fregante) [inline .xo-config.json:11]: Too much noise on an existing codebase

File: source/features/use-first-commit-message-for-new-prs.tsx [hunk 3]
```diff
@@ -6,12 +6,7 @@ import * as textFieldEdit from 'text-field-edit';
 import features from '.';
 import looseParseInt from '../helpers/loose-parse-int';
 
-// TODO [2022-05-01]: Drop GHE code and merge function back in init
```
Comment (fregante) [inline source/features/use-first-commit-message-for-new-prs.tsx:9]: No merge needed; init is large enough

File: source/globals.d.ts [hunk 4]
```diff
@@ -1,6 +1,7 @@
 /* eslint-disable @typescript-eslint/consistent-indexed-object-style */
 
 type AnyObject = Record<string, any>;
+// TODO: Fix this type, there's no such `delegate` global type
 type Deinit = delegate.Subscription | Observer | MutationObserver | ResizeObserver | IntersectionObserver | VoidFunction;
```
Comment (fregante) [inline source/globals.d.ts:5]: `delegate.Subscription` and therefore `Deinit` are `any` due to #5422 @cheap-glitch

File: source/globals.d.ts [hunk 5]
```diff
@@ -1,6 +1,7 @@
 /* eslint-disable @typescript-eslint/consistent-indexed-object-style */
 
 type AnyObject = Record<string, any>;
+// TODO: Fix this type, there's no such `delegate` global type
```
Comment (FloEdelmann) [inline source/globals.d.ts:4]:
```suggestion
// TODO: Fix this type (there's no such `delegate` global type) and enable the @typescript-eslint/no-redundant-type-constituents rule
```

**Comment by fregante** [comment]
All PRs merged, 20 bug issues open. I haven’t seen that in a while ☺️ 

New release coming in 10-15 hours. If I forget, please someone do it.

**Pull request merged**